src/Orleans/Serialization/TypeUtilities.cs

```diff
@@ -266,12 +266,7 @@ private static bool AreInternalsVisibleTo(Assembly fromAssembly, Assembly toAsse
             // Check InternalsVisibleTo attributes on the from-assembly, pointing to the to-assembly.
             var serializationAssemblyName = toAssembly.GetName().FullName;
             var internalsVisibleTo = fromAssembly.GetCustomAttributes<InternalsVisibleToAttribute>();
-            if (internalsVisibleTo.All(_ => _.AssemblyName != serializationAssemblyName))
-            {
-                return true;
```

Comment:
I did invert the condition actually. Your right in that I could just invert the return value though.

Re# normally recommends collapsing conditional statements with boolean return values into a single return statement so I just went ahead and jumped straight to the final code.